### DISCUSSION TITLE
China Piracy

### DISCUSSION THREAD

**Discussion by Bammede** [mylinuxforwork/dotfiles#76] [General]
Your project is in China too run by someone else  ;-)

https://gitee.com/KenZhv/hl4w-dotfiles

**Comment by FurtherSecrets24680** [comment]
![image](https://github.com/user-attachments/assets/f7e72f2d-d4da-4cc9-8e0b-6a3ba7dc0bb9)

Plus they didn't even include this repo in the credits!
BECAUSE this section is also stolen from the official one!

![image](https://github.com/user-attachments/assets/49602df6-2af1-49a6-9cd7-64b82ac30a22)

Lol, they forgot to edit this out.

I guess many good projects from GitHub is being stolen, modified and reuploaded on other websites.

**Reply by mylinuxforwork** [reply]
I have no worries about it. It's open source and public. But thanks for sharing ;-)
